packages/vision/src/helpers.js

```diff
@@ -0,0 +1,197 @@
+    callback(null, {
+      content: image.toString('base64')
+    });
+    return;
+  }
+
+  // File exists on disk.
+  if (image.source && image.source.filename) {
+    fs.readFile(image.source.filename, {encoding: 'base64'}, (err, blob) => {
+      if (err) {
+        callback(err);
+        return;
+      }
+      callback(null, {content: blob.toString('base64')});
+    });
+    return;
+  }
+
+  // No other options were relevant; return the image with no modification.
+  callback(null, image);
+  return;
+};
+
+
+/**
+ * Return a method that calls annotateImage asking for a single feature.
+ *
+ * @param {Number} featureValue - The feature being requested. This is taken
+ *   from the Feature.Type enum, and will be an integer.
+ *
+ * @return {Function} - The function that, when called, will call annotateImage
+ *   asking for the single feature annotation.
+ */
+var _createSingleFeatureMethod = featureValue => {
+  return function(image, options) {
+    return this.annotateImage({
+      image: image,
+      features: [{type: featureValue}],
+    }, options);
+  };
+};
+
+
+/**
+ * Return a dictionary-like object with helpers to augment the Vision
+ * GAPIC.
+ *
+ * @param {string} apiVersion - The API version (e.g. "v1")
+ *
+ * @return {Object} - An object with keys and functions which are placed
+ *   onto the pure GAPIC.
+ */
+module.exports = apiVersion => {
+  var methods = {};
+
+  /**
+   * Annotate a single image with the requested features.
+   *
+   * @param {Object=} request
+   *   A representation of the request being sent to the Vision API.
+   * @param {Object=} request.image
+   *   A dictionary-like object representing the image. This should have a
+   *   single key (`source`, `content`).
+   *
+   *   If the key is `source`, the value should be another object containing
+   *   `image_uri` or `filename` as a key and a string as a value.
+   *
+   *   If the key is `content`, the value should be a Buffer.
+   * @param {Array} request.features
+   *   An array of the specific annotation features being requested.
+   * @param {Object=} options
+   *   Optional parameters. You can override the default settings for this
+   *   call, e.g, timeout, retries, paginations, etc. See [gax.CallOptions]
+   *   {@link https://googleapis.github.io/gax-nodejs/global.html#CallOptions}
+   *   for the details.
+   * @param {function(?Error, ?Object)=} callback
+   *   The function which will be called with the result of the API call.
+   *
+   *   The second parameter to the callback is an object representing
+   *   [AnnotateImagesResponse]{@link AnnotateImagesResponse}.
+   * @return {Promise} - The promise which resolves to an array.
+   *   The first element of the array is an object representing
+   *   [AnnotateImagesResponse]{@link AnnotateImagesResponse}.
+   *   The promise has a method named "cancel" which cancels the ongoing
+   *   API call.
+   *
+   * @example
+   * var vision = require('@google-cloud/vision').v1();
```

Comment:
Is it important to use the `v1()` here, or can we just use "require('@google-cloud/vision')()", since they are the same?

If we don't need the `v1()`, you can actually omit this line. Our docs script will always define and show the instantiation of "var moduleName = require('@google-cloud/module-name')` in the docs. This way, all methods are free to use the `moduleName` variable, without having to show how to instantiate the module in each block.